tests/test_examples.py

```diff
@@ -377,6 +377,28 @@ def _is_micromamba(path) -> bool:
     return name == StandaloneExe.MAMBA
 
 
+@pytest.fixture(params=["linux-aarch64"])
+def platform_conda_exe(request, tmp_path) -> Tuple[str, Path]:
+    platform = request.param
+    tmp_env = tmp_path / "env"
+    subprocess.check_call(
+        [
+            sys.executable,
+            "-mconda",
```

Comment:
```suggestion
            "-m",
            "conda",
```

I assume you mean this? If so, how did the test pass?

Reply:
They are equivalent.